test/suite/lib.rs

```diff
@@ -63,74 +63,100 @@ async fn assert_balance(
     Ok(())
 }
 
+async fn setup_test_accounts(
```

Comment:
This isn't nice to use because whatever you do you'll have to "unpack" the vector that is the result. Instead I would have a `setup_test_account` (singular). The usage would change from

```
let envs = setup_test_accounts(&ledger_env, amount, 2);
let env1 = envs[0].clone();
let env2 = envs[1].clone();
```

to

```
let env1 = setup_test_account(&ledger_env, amount);
let env2 = setup_test_account(&ledger_env, amount);
```

The additional bonus is that it's more flexible allowing to create test accounts with different amount of tokens.